### ISSUE TITLE
JS: Objects

### ISSUE THREAD

**Issue by paramsiddharth** [girlscript/winter-of-contributing#246]
# Objects
Objects in JavaScript.

## Directory
```
Topics/5.Syntax/6.Functions_And_Scope
```

## Description
* About JavaScript objects.
* How to define them?
* Getters and setters.
* What are keys and values?

**Comment by siddhi-244** [comment]
Can I try it

**Comment by siddhi-244** [comment]
In terms of documentation

**Comment by paramsiddharth** [comment]
@siddhi-244 Sure! You may, though I'd encourage you to work on one issue at a time. Best of luck!

**Comment by siddhi-244** [comment]
> @siddhi-244 Sure! You may, though I'd encourage you to work on one issue at a time. Best of luck!

I can do it together

**Comment by aniketpathak028** [comment]
I want to work on this issue, can I try it?

**Comment by AbhinandanAdhikari** [comment]
Sir, I want to work on it.

**Comment by sharia-farooqui** [comment]
Can I work on this issue?

**Comment by gurpreet-legend** [comment]
Sir, can I work on this issue ?
In the Documentation domain .

**Comment by faisal-kn** [comment]
Sir, I am quite experienced with javascript. Can I work on this issue ?

**Comment by siddhi-244** [comment]
this issue was assigned to me but i see a contribution already?

**Comment by paramsiddharth** [comment]
> this issue was assigned to me but i see a contribution already?

Yes, Siddhi. As I previously mentioned, one issue may be assigned to multiple contributors at once. Someone finished a PR and got it approved before you. You may now choose to extend their contribution or contribute a different format if you wish to continue your contribution to this issue, or move on to another issue.

**Comment by Divyanshigarg** [comment]
sir I too wants to contribute in this issue. Please assign me if possible.
